src/excmds.ts

```diff
@@ -175,8 +175,8 @@ ALL_EXCMDS = {
 
 /** @hidden **/
 //#background
-export async function getNativeVersion(): Promise<void> {
-    Native.getNativeMessengerVersion()
+export async function getNativeVersion(): Promise<number> {
```

Comment:
Calling it a number seems dangerous, e.g. 0.19 > 0.2 should be true, and 0.19.1 is not a number

Reply:
You're right, I just blindly copied what I found in https://github.com/tridactyl/tridactyl/blob/27572a59149ed7a3d423a531c1af664d82d20c1c/src/lib/native.ts#L81-L91 . I guess I'll update both :)